src/org/exist/backup/Main.java

```diff
@@ -127,6 +127,9 @@
     private static final Argument<File> backupOutputDirArg = fileArgument("-d", "--dir")
             .description("specify the directory to use for backups.")
             .build();
+    private static final Argument<Boolean> backupDeduplicateBlobs = booleanArgument("--deduplicate-blobs")
```

Comment:
but what does it do?

Reply:
really storing a binary file once in the ZIP file? nice! but it needs a note that the backup file is not backwards compatible, if any relevance.

Reply:
Indeed. The `__contents__.xml` files have a version number, and it will report if the backups are not compatible.